apps/client/src/routes/dashboards/dashboards-index/components/migration.tsx

```diff
@@ -0,0 +1,54 @@
+import { memo } from 'react';
+import {
+  ExpandableSection,
+  Box,
+  Button,
+  ColumnLayout,
+} from '@cloudscape-design/components';
```

Comment:
Use default imports for Cloudscape components to properly tree-shake the library.

For example:

```
import Box from '@cloudscape-design/components/box`;
import ExpandableSection from '@cloudscape-design/components/expandable-section';
```

See https://cloudscape.design/get-started/integration/using-cloudscape-components for reference.

Reply:
Makes sense, updated.